CMakeLists.txt

```diff
@@ -247,22 +249,27 @@ endif()
 #compile as C++/static CRT on Visual Studio and as C99 on UNIX
 #//////////////////////////
 
+set(NCO_MSVC_USE_MT yes CACHE BOOL "Use MT flags when compiling in MSVC")
+
 if (MSVC)
   add_definitions( "/D_CRT_SECURE_NO_WARNINGS /D_CRT_NONSTDC_NO_DEPRECATE" )
-  message("-- Building with static runtime library")
-  set(CMAKE_CXX_FLAGS_DEBUG "/D_DEBUG /MTd /Zi /Ob0 /Od /RTC1")
```

Comment:
Using the default CMake values and modifying below allows the user to append to these from the command line.